### DISCUSSION TITLE
Web support

### DISCUSSION THREAD

**Discussion by JosephTLyons** [zed-industries/zed#26195] [Feature Requests]
A quick note on platform support can be found here:
- https://github.com/zed-industries/zed/issues/5391

**Comment by xnousnow** [comment]
Will the layout of the web version be the same as the default version or a bit edited?

**Comment by hovsater** [comment]
Cross platform support is still very much on our radar and we’re dedicated to support both Windows and Linux in the future. As we've previously stated, we aren't taking feedback on the roadmap for platform support and as such we've decided to lock this issue. Any update regarding this will be noted on our website.
